scripts/zkai_attention_kv_sixteen_head_air_private_softmax_table_lookup_gate.py

```
@@ -0,0 +1,1091 @@
+    }
+    payload["gate_commitment"] = gate_commitment_for(payload, receipt)
+    cases = mutation_cases_for(payload)
+    payload["mutation_cases"] = cases
+    payload["mutations_checked"] = len(cases)
+    payload["mutations_rejected"] = sum(1 for case in cases if case["rejected"])
+    payload["all_mutations_rejected"] = payload["mutations_checked"] == payload["mutations_rejected"]
+    validate_payload(payload)
+    return payload
+
+
+def mutate_payload(payload: dict[str, Any], name: str) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Too many branches (33 > 12)

(PLR0912)